lighthouse-core/lib/i18n/i18n.js

```diff
@@ -64,6 +64,8 @@ const UIStrings = {
   columnRequests: 'Requests',
   /** Label for a column in a data table; entries will be the number of kilobytes transferred to load a set of files. */
   columnTransferSize: 'Transfer Size',
+  /** Label for a column in a data table; entries will be the names of arbitrary objects. */
```

Comment:
> entries will be the names of arbitrary objects

seems like we could do better than this, this just kicks the can from "what does 'name' refer to?" down to "what does 'object' refer to?" :P

Reply:
better?